### PR TITLE
Feat/reoccuring job

### LINKED ISSUE #104: Dialog to specify recuring jobs

### PR DESCRIPTION
**Pull request by soofstad**
- Component for generating a limited and simplified "cron job syntax"
- Functionality for registering and removing cronJobs on a SimConfig

closes #104

### CODE REVIEW

File: web/custom-plugins/forecast-of-response/src/components/ReoccurringJob.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+      </InputWrapper>
+
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'space-around',
+          margin: '10px',
+        }}
+      >
+        <Button color={'secondary'} onClick={() => close()}>
```
Comment (KristianKjerstad) [inline web/custom-plugins/forecast-of-response/src/components/ReoccurringJob.tsx:127]: when testing locally, it was difficult to understand the difference between the close and remove button.

File: web/custom-plugins/forecast-of-response/src/components/ReoccurringJob.tsx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+        </Button>
+        <Button
+          color="danger"
+          variant="outlined"
+          onClick={() => {
+            removeJob()
+            close()
+          }}
+        >
+          Remove
```
Comment (KristianKjerstad) [inline web/custom-plugins/forecast-of-response/src/components/ReoccurringJob.tsx:138]: I think the remove button should be disabled if there are no recurring job defined

**Review by KristianKjerstad — APPROVED** [review]
comment

for interval: weekly, we could have some text to say which day the job will run - monday/tuesday/... (or let the user specify the day himself)

same for interval monthly - can have text to inform what date (1. , 2. , 3. etc) the job will run on

image

File: web/custom-plugins/forecast-of-response/src/components/ReoccurringJob.tsx (hunk 2 above)
Comment (soofstad) [reply web/custom-plugins/forecast-of-response/src/components/ReoccurringJob.tsx:146]: fixed

**Comment by soofstad** [comment]
> comment
> 
> for interval: weekly, we could have some text to say which day the job will run - monday/tuesday/... (or let the user specify the day himself)
> 
> same for interval monthly - can have text to inform what date (1. , 2. , 3. etc) the job will run on
> 
> image

fixed

**Pull request merged**